### ISSUE TITLE
Windows install fails on dex deployment

### ISSUE THREAD

**Issue by kkaempf** [epinio/epinio#672]
* epinio (built from source) @ c7afb19

* Windows 10 Home, Kubernetes via Rancher Desktop

* Running `epinio install` in PowerShell

Installation fails with
```
panic: Deployment dex failed with error: Failed to extract embedded file:  - file does not exist


goroutine 52 [running]:
github.com/epinio/epinio/internal/cli/clients.(*InstallClient).Install.func1(0xc000254150, 0x24ca9b0, 0xc000034050, 0x24d5670, 0xc00044b400, 0x24d9df8, 0xc00020d0b0, 0xc00020d094)
        /abuild/projects/rancher/epinio/epinio/internal/cli/clients/install_client.go:168 +0x23f
created by github.com/epinio/epinio/internal/cli/clients.(*InstallClient).Install
        /abuild/projects/rancher/epinio/epinio/internal/cli/clients/install_client.go:165 +0x10e5
```

**Comment by kkaempf** [comment]
The `build-windows` target does not run `embed_files`. Running `make embed_files` _before_ running `make build-windows` fixes this problem.

Keeping this bug open to get this dependency expressed in the Makefile.

**Comment by andreas-kupries** [comment]
This feels like going back a bit again on our desire of having the low-level targets independent of each other and using higher level targets which combine low-level operations as desired/needed.

At the moment all the `build-(platform)` targets are low-level, and there are two highlevel targets, `build`, and `build-all`.
Where `build` is `embed_files` + `build-amd64`, and `build-all` is `embed_files` + all `build-()` targets.

Or are you asking for a new highlevel target combining `embed_files` and `build-windows`, name to be determined ?

**Comment by andreas-kupries** [comment]
See #671 for a commit doing the latter.

**Comment by kkaempf** [comment]
Confirmed that the Dex problem is fixed in #671.
